### DISCUSSION TITLE
Backup for Restore Flipper

### DISCUSSION THREAD

**Discussion by expressaccount** [akopachov/flipper-zero_authenticator#248] [Q&A] [answered]
I have read the FAQ about the backup. But I still have a question. If I create the Flipper backup via qFlipper. Are 2FAs then backed up? I'm not asking because of the loss of the Flipper, but in case i have to resetmy Flipper and don't want to re-enter everything.

**Comment by akopachov** [comment, marked as answer]
You are correct. Assuming that never ever anything happens with your original Flipper that brick it - it is enough to backup just the `/ext/apps_data/totp/totp.conf` file.

**Reply by expressaccount** [reply]
Thank you!
